frontend/packages/kubevirt-plugin/src/selectors/vmi/selectors.ts

```diff
@@ -17,3 +17,11 @@ export const getVMIDisks = (vmi: VMIKind): VMIKind['spec']['domain']['devices'][
   vmi && vmi.spec && vmi.spec.domain && vmi.spec.domain.devices && vmi.spec.domain.devices.disks
     ? vmi.spec.domain.devices.disks
     : [];
+
+export const getVMIConditionByType = (
```

Comment:
we should either rename this to `getVMIConditionsByType` (Conditions as plural since you are using `filter` and returning array) or keep the name and change return type and use `find`

Reply:
renamed